packages/server-boilerplate/src/orchestrator/api/ApiBuilder.ts

```diff
@@ -76,6 +81,12 @@ export class ApiBuilder {
      * Add singletons to be attached to req for every route
      */
     this.app.use((req: Request, res: Response, next: NextFunction) => {
+      if (options.attachModels) {
+        winston.warn(
+          "Best practices say orchestrator servers shouldn't access the db directly, are you sure you need req.models?",
+        );
+        req.models = this.models;
+      }
```

Comment:
I do really like this, I think it's great to include these kinds of conventions in the code itself.

I worry this one might get a tad noisy, but happy to see how it goes 👍